scripts/release/issue-list.sh

```diff
@@ -55,7 +62,9 @@ function milestone_issues()
     if [[ "$line" == *"(task)"* ]]; then tasks="$tasks\n$line"; fi
   done <<< "$issue_list"
 
-  echo -e "$features\n$bugs\n$tasks"
+  print_content_with_nonempty_issues "$features"
+  print_content_with_nonempty_issues "$bugs"
+  print_content_with_nonempty_issues "$tasks"
```

Comment:
I think we should do this in a loop
for i in $features $bugs $tasks
do
    print_content_with_nonempty_issues $i
done